extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -74,6 +75,11 @@ export class AmpAd extends AMP.BaseElement {
       const slotId = window.ampAdSlotIdCounter++;
       this.element.setAttribute('data-amp-slot-index', slotId);
 
+      if (this.element.getAttribute('data-multi-size')
+          && Object.keys(getAllParentAdContainers(this.element)).length != 0) {
```

Comment:
Changing this to return an array instead of an object will make this statement much cleaner.